build.cmd

```diff
@@ -0,0 +1,16 @@
+<Project>
```

Comment:
It's not a CMD file, so I wouldn't call it that. I think `build.msbuild` or even `Microsoft.jQuery.Unobtrusive.Validation.msbuild` is fine, then have a build.cmd that runs it.